### DISCUSSION TITLE
Animating Patterns

### DISCUSSION THREAD

**Discussion by gbishop** [UNC-Project-Open-AAC/OS-DPI#230] [Ideas]
In an attempt to make Patterns easier to understand I am playing with an animation feature. Please check out this video and tell me if you think something like this could help and how I might improve it.

It is stepping through the Pattern and activating each Group as it encounters them. 

[Screencast from 2023-11-09 09-23-10.webm](https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/198012/54f97f4c-d447-4993-9739-f942074ab6d6)

**Comment by aetkie** [comment]
The animate feature is helpful. But the scanning is fast. How do I slow it down?

**Comment by gbishop** [comment]
If I make it go slower it could take a long time on a really big grid.
There should be preferences somewhere in the interface.
